src/steppy/pytorch/models.py

```diff
@@ -5,11 +5,13 @@
 import numpy as np
 import torch
 import torch.nn as nn
+# from torch.nn import DataParallel
+from .parallel import DataParallelModel as DataParallel
```

Comment:
@taraspiotr since we are overwriting the model anyways lets just have it in the `src/models.py` for clarity